docs/ethereum-section/wrap-document-eth.md

```diff
@@ -45,7 +45,7 @@ wrapped-documents
   |-- certificate-2.json
 ```
 
-In the `wrapped-documents` directory, you will find the wrapped document which can be sent to the recipient later, once the `merkleRoot` has been issued to the document store.
+In the `wrapped-documents` directory, you will find the wrapped documents that will be issued on the document store using `merkleRoot` in the next article. 
```

Comment:
```suggestion
In the `wrapped-documents` directory, you will find the wrapped documents that will be issued to the document store in the next article. 
```

On a side note, I noticed that we are inconsistent in the phrasing for `issued on the document store` or `issued to the document store`, let's standardize this to the latter.

Reply:
After searching for `on document store` and `on the document store`, only one more article was found & updated along with this one.